### PR TITLE
[Release v2.5.0][PTQ] Add status bar for FBC, BC and statistics collection 

### PR DESCRIPTION
**Pull request by kshpv**
### Changes

Add `tqdm` status bars.
image

### Reason for changes

Improve UX

### Related tickets

...

### CODE REVIEW

**Review by KodiaqQ — APPROVED** [review]
Waiting for an approval from @MaximProshin

**Review by MaximProshin — APPROVED** [review]

**Comment by MaximProshin** [comment]
non-functional and I assume ok to merge it

**Pull request merged**
